lib/dogapi/facade.rb

```diff
@@ -44,6 +44,7 @@ def initialize(api_key, application_key=nil, host=nil, device=nil, silent=true,
       @embed_svc = Dogapi::V1::EmbedService.new(@api_key, @application_key, silent, timeout, @datadog_host)
       @screenboard_svc = Dogapi::V1::ScreenboardService.new(@api_key, @application_key, silent, timeout, @datadog_host)
       @monitor_svc = Dogapi::V1::MonitorService.new(@api_key, @application_key, silent, timeout, @datadog_host)
+      @synthetic_svc = Dogapi::V1::SyntheticService.new(@api_key, @application_key, silent, timeout, @datadog_host)
```

Comment:
remove `app/integration/synthetic.rb` as this  looks like it got pulled in from work unrelated to this repository?